Gemfile.lock.release

```diff
@@ -772,9 +797,9 @@ GEM
       json (~> 2.3)
       more_core_extensions
       query_relation
-    manageiq-appliance_console (6.0.0)
-      activerecord (>= 4.2.2)
-      activesupport (>= 4.2.2)
+    manageiq-appliance_console (7.0.1)
+      activerecord (~> 6.0.4, >= 6.0.4.1)
+      activesupport (~> 6.0.4, >= 6.0.4.1)
```

Comment:
As discussed (somewhere, but I can't find the discussion now 😅 ), This should be manageiq-appliance_console 6.1.1 in order to not bring in the major version.

Reply:
https://github.com/ManageIQ/manageiq/pull/21447